.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "agent/lib/libjpeg-turbo"] 
```

Comment:
I'd rather not go the route of submodules because AFAIK the cloning process is somewhat more involved.